vcd_work_terminate() calls pthread_join() on a static thread.
Do not call vcd_work_terminate() in close_dumpfile() because it was already
called before, leading to a second call to pthread_join() on the same pthread,
which is undefined behaviour.

--- vpi/sys_lxt2.c.orig	2020-05-13 00:25:45.070186668 +0200
+++ vpi/sys_lxt2.c	2020-05-13 00:26:09.721185704 +0200
@@ -472,7 +472,6 @@
 
 static void *close_dumpfile(void)
 {
-      vcd_work_terminate();
       lxt2_wr_close(dump_file);
       dump_file = NULL;
       return NULL;
